### ISSUE TITLE
You shouldn't be able to send a Yo! message to server without You! CorDapps installed

### ISSUE THREAD

**Issue by wawrzek** [roger3cev/yo-cordapp#3]
Now you can send a Yo to a node without Yo installed and you get 'success'. 

Yo fails only if sent to a not active node. That's my guess, because I don't see anything in logs, and webpage "spins" indefinitely.

**Comment by roger-that-dev** [comment]
This is a feature not a bug ;) All nodes have the flow to receive a transaction but without the yo-app installed you can't serialise the Yo, thus it fails.

We need to add a filter so you can only send Yo's to nodes that have the app installed.

Cheers

**Comment by wawrzek** [comment]
Agreed. The report tiles probably should be: **You shouldn't be able to send a Yo! message to server without You! CorDapps installed**. BTW does it mean we would have a kind of 'Yo!' business network?
